org.learningequality.Kolibri.json

```diff
@@ -0,0 +1,72 @@
+{
+  "app-id": "org.learningequality.Kolibri",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "19.08",
+  "sdk": "org.freedesktop.Sdk",
+  "finish-args": [
+    "--share=network",
+    "--filesystem=~/.kolibri:create"
```

Comment:
One reason for mounting it here was to make it easier for the user to directly interact with the files in it if needed (e.g. we sometimes ask people to pull a log file from in here, or copy content files in or out, or edit a config file. It looks as if the sandbox location for the `persist` option would still be accessible externally, though, under `~/.var/app/org.learningequality.Kolibri/.kolibri` -- so it would still be possible to have people access that directly, but we'd need to have special instructions for flatpak users.

Reply:
(I've made the change for now, but leaving this unresolved in case there are any ideas on the above)

Reply:
Yeah, this would be inside that directory.